pkg/kopia/command/parse_command_output.go

```diff
@@ -327,6 +331,93 @@ func parseKopiaProgressLine(line string, matchOnlyFinished bool) (stats *Snapsho
 	}
 }
 
+// SnapshotRestoreStats is a container for stats parsed from the output of a
+// `kopia snapshot restore` command.
+type SnapshotRestoreStats struct {
+	FilesProcessed  int64
+	SizeProcessedB  int64
+	FilesTotal      int64
+	SizeTotalB      int64
+	ProgressPercent int64
+}
+
+// SnapshotStatsFromSnapshotRestore parses the output of a `kopia snapshot
+// restore` execution for a log of the stats for that execution.
```

Comment:
sorry, I was not able to understand this comment properly, especially the part `execution for a log of the stats for that execution.`.

Reply:
The comment was done in same styling (wording) as in similar function: 
https://github.com/kanisterio/kanister/pull/2776/files#diff-00217f95488de292da931fcdc8bdc46e611d503b26e17e13119cb4ce04548892R214-R215

In both cases it means that we have output of some command, we are parsing it and trying to find stats logged by command.

Possible rephrasing (for both cases) :
// XXX parses the output of 'ZZZZZ' line-by-line in search of progress statistics.
// It returns nil if no statistics are found, or the most recent statistic if multiple are encountered.

WDYT ?

Reply:
sounds good.

Reply:
Replaced for both `SnapshotStatsFromSnapshotCreate` and `RestoreStatsFromRestoreOutput`